scss/bitstyles/organisms/page-header/_index.scss

```diff
@@ -0,0 +1,66 @@
+@use '../../tools/media-query';
+@use '../../tools/classname';
+@use '../../tools/design-token';
+
+#{classname.get($classname-items: 'page-header', $layer: 'organism')} {
+  padding-top: var(design-token.get('size', 'm'));
+  background-color: var(design-token.get('color', 'grayscale', 'light-3'));
+}
+
+.o-page-header__content {
```

Comment:
Each of these classnames also needs to be generated, same as in line 5 — perhaps instead, this is an ideal place to make use of sass nested classnames? e.g.

```scss
&__content {}
```
(and nested under the root classname above)